### ISSUE TITLE
Have install.sh check for ansible installation and install it if not present

### ISSUE THREAD

**Issue by mdvickst** [quipucords/quipucords#1238]
## Specify type:
- Enhancement

### Priority:
- Medium
___

## Description:
It would be nice if install.sh could check if ansible is installed and install it if not. One pilot customer didn't see the prerequisite to install Ansible so the install.sh script failed. It would be nice to have it fail gracefully if Ansible is not installed or better yet have it install Ansible based on the version of RHEL/Centos that is running.

___

## Acceptance Criteria:

- [ ] Verify that install.sh fails gracefully if Ansible is not installed

___

**Comment by chambridge** [comment]
I don't think we can install Ansible as part of the `install.sh` script as this requires `sudo` or `root` authority for most systems; which would mean the user would need to run `sudo install.sh` and that would escalate privilege for all of the Ansible playbook roles instead of selectively.

Should be a simple `command -v ansible` check in the install script and provide the appropriate error message.

**Comment by chambridge** [comment]
@mdvickst 

Trying to understand how much you want the install script to do.
If Centos or RHEL6 should we be checking for and potentially installing EPEL?
For RHEL7 should there be a check to see if Extra repo is enabled?

**Comment by chambridge** [comment]
Currently planning on letting the above PR go in, but just not closing this issue if you want us to add the complexity to the install.sh script.

**Comment by chambridge** [comment]
Based on the outcome of the above conversation we will prioritize. Catching missing prereq at a minimum now.

**Comment by mdvickst** [comment]
I think addition of EPEL/Extras repos in addition to the yum install of ansible would be great. I do think we should have some way of error checking that in case they are not connected to the internet etc. but it would be ideal if the install.sh could start to finish install qpc, maybe we could even remove the docs section to install ansible if they are doing the scripted install.
